src/controllers/handlers/scene-admin-handlers/remove-scene-admin-handler.ts

```diff
@@ -0,0 +1,74 @@
+import { EthAddress } from '@dcl/schemas'
+import { HandlerContextWithPath, InvalidRequestError, UnauthorizedError } from '../../../types'
+import { validate } from '../utils'
+
+export async function removeSceneAdminHandler(
+  ctx: Pick<
+    HandlerContextWithPath<'sceneAdminManager' | 'sceneFetcher' | 'logs' | 'config' | 'fetch', '/scene-admin'>,
+    'components' | 'url' | 'params' | 'verification' | 'request'
+  >
+) {
+  const {
+    components: { logs, sceneFetcher, sceneAdminManager },
+    request,
+    verification
+  } = ctx
+
+  const logger = logs.getLogger('remove-scene-admin-handler')
+
+  const { getPlace, hasLandPermission, hasWorldPermission } = sceneFetcher
+
+  if (!verification?.auth) {
+    throw new UnauthorizedError('Authentication required')
+  }
+
+  const payload = await request.json()
+
+  if (!payload.admin) {
+    logger.warn(`Invalid scene admin payload`, payload)
+    throw new InvalidRequestError(`Invalid payload`)
+  }
+
+  const { parcel, hostname, realmName } = await validate(ctx)
+  const isWorlds = hostname.includes('worlds-content-server')
+  const authAddress = verification.auth.toLowerCase()
+
+  if (!EthAddress.validate(payload.admin)) {
+    throw new UnauthorizedError('Invalid admin address')
+  }
+
+  const place = await getPlace(isWorlds, realmName, parcel)
+  if (!place) {
+    throw new InvalidRequestError('Place not found')
+  }
+
+  const hasPermission =
+    (isWorlds
+      ? await hasWorldPermission(authAddress, place.world_name!)
+      : await hasLandPermission(authAddress, place.positions)) ||
+    (await sceneAdminManager.isAdmin(place.id, authAddress))
+
+  if (!hasPermission) {
+    logger.warn(`User ${authAddress} is not authorized to remove admins for entity ${place.id}`)
+    throw new UnauthorizedError('Only scene admins or the owner can remove admins')
+  }
+
+  const isAdminOwner = isWorlds
+    ? await hasWorldPermission(payload.admin, place.world_name!)
+    : await hasLandPermission(payload.admin, place.positions)
```

Comment:
how come this analyze if the `payload.admin` is the owner? shouldn't we call `sceneAdminManager.isAdmin`?

Reply:
I don't get what will happen if payload.admin is the land/world owner instead of a simple admin

Reply:
I changed the name to `IsOwnerOrAdmin`, I hope it's clear enough